src/14_cal.py

```diff
@@ -33,15 +33,15 @@
 
 
 def render_calendar():
-    if len(sys.argv) > 3:
-        print("Script to print a calendar for a given month.")
-        print("If no options are specified, prints the current month")
-        print("usage: app_name <month> <optional-year>")
-    else:
+    try:
```

Comment:
Good use of try and except instead using if statements. Ideal programming practice